app/utils/normalize-device-id.js

```diff
@@ -7,7 +7,21 @@ export function normalizeDeviceId(id) {
 
   if (length === 6) {
     if (RE_HEXDEC.test(id)) {
-      return `FLR${id}`;
+      // Perform guestimates based on 6-digit device ID.
+      // In case device ID is in the range D00000 to DFFFFF it is most likely a FLARM ID
+      if (parseInt(id, 16) >= 0xd00000 && parseInt(id, 16) <= 0xdfffff) {
```

Comment:
can you extract the `parseInt(id, 16)` to a variable so that we only need to perform it once?